### DISCUSSION TITLE
How to exclude variables from plot.subsample?

### DISCUSSION THREAD

**Discussion by simontrlifaj** [kogalur/randomForestSRC#223] [Q&A] [answered]
When running plot.subsample, is there a way exclude some variables from the plot? I am training a random forest, and I am only interested in the performance of a few variables. Alternatively, can I exclude them already when calculating the confidence intervals with subsample?

**Comment by ishwaran** [comment, marked as answer]
Currently no.  However it is easy to add this option and we will do so in the upcoming release of the package.  It's a great suggestion.

Do not however remove the variables when calculating the confidence intervals as this alters the statistical inference.If you need a work around let me know.  It would require modifying the `plot.subsample` function.

**Reply by simontrlifaj** [reply]
Thanks for the quick reply! I might need a workaround. I was thinking of exporting the statistics from the object created by the `subsample` function. I will probably need them anyway. Is there an easy way to do that?

**Reply by ishwaran** [reply]
There is a helper function `extract.subsample` that takes the subsample object and parses it and returns useful summary information.  This function is used in `plot.subsample`.  So that would probably be the function you should use.  That being said, I will also modify `plot.subsample` to invisibly return the box plot data that is being plotted since that could be very useful.
